src/components/Modals/SubscriptionsModal/SubscriptionsModal.tsx

```diff
@@ -49,7 +49,7 @@ export const SubscriptionsModal: React.FC<Props> = ({
                   height={36}
                   className={s.modal__content__avatar}
                 />
-                <p>URLProfiele</p>
+                <p>URLProfielgwegwergrgety45h454545yh45y</p>
```

Comment:
is it ok? URLProfielgwegwergrgety45h454545yh45y

Reply:
its for test) will fix it